pkg/parameters/parameters.go

```diff
@@ -0,0 +1,16 @@
+// argument command struct based on ocm cli
+// https://github.com/openshift-online/ocm-cli/blob/main/pkg/arguments/arguments.go
+
+package parameters
+
+type ValidateEgress struct {
+	VpcSubnetID string
+}
+
+type ValidateDns struct {
+	VpcId string
+}
+
+type ValidateByoVpc struct {
+	//	todo
+}
```

Comment:
Why are these defined in a separate package? Is there any reason not to define them right next to their related functions in the `cloudclient` package directly?